# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1661673204 -7200
# Parent  243f1e5c5abe6a59f26772386c7c9e51d910ddd8
Branding 2: Rename net.thunderbird.Thunderbird.appdata.xml to eu.betterbird.Betterbird.appdata.xml.

diff --git a/mail/branding/betterbird/net.thunderbird.Thunderbird.appdata.xml b/mail/branding/betterbird/eu.betterbird.Betterbird.appdata.xml
rename from mail/branding/betterbird/net.thunderbird.Thunderbird.appdata.xml
rename to mail/branding/betterbird/eu.betterbird.Betterbird.appdata.xml
--- a/mail/branding/betterbird/net.thunderbird.Thunderbird.appdata.xml
+++ b/mail/branding/betterbird/eu.betterbird.Betterbird.appdata.xml
@@ -11,23 +11,23 @@
   </description>
   <categories>
     <category>Calendar</category>
     <category>Email</category>
     <category>Office</category>
   </categories>
 
   <url type="homepage">https://www.betterbird.eu/</url>
-  <url type="bugtracker">https://bugzilla.mozilla.org/</url>
+  <url type="bugtracker">https://github.com/Betterbird/thunderbird-patches</url>
   <url type="faq">https://www.betterbird.eu/faq/</url>
-  <url type="help">https://support.mozilla.org/products/thunderbird/</url>
+  <url type="help">https://www.betterbird.eu/support/</url>
   <url type="donation">https://www.betterbird.eu/donate/</url>
-  <url type="translate">https://www.thunderbird.net/en-US/get-involved/#translation</url>
+  <url type="translate">https://github.com/Betterbird/thunderbird-patches</url>
 
-  <project_group>Mozilla</project_group>
+  <project_group>Betterbird</project_group>
   <project_license>MPL-2.0</project_license>
   <developer_name>Betterbird Project</developer_name>
 
   <icon type="remote" width="256" height="256">https://www.betterbird.eu/media/img/betterbird-256.png</icon>
 
   <mimetypes>
     <mimetype>message/rfc822</mimetype>
     <mimetype>x-scheme-handler/mailto</mimetype>
